src/utils.ts

```diff
@@ -44,6 +44,44 @@ export function prefixStorage<T extends StorageValue>(
       // Remove Prefix
       .then((keys) => keys.map((key) => key.slice(base.length)));
 
+  if (typeof storage.getItems === "function") {
+    nsStorage.getItems = <U extends T>(
+      items: (string | { key: string; options?: any })[],
+      commonOptions?: any
```

Comment:
both `options` and `commonOptions` are `TransactionOptions`. so you may as well use the real type here instead of `any`